src/Bundle/DependencyInjection/Configuration.php

```diff
@@ -40,6 +40,7 @@ private function addDriversSection(ArrayNodeDefinition $node): void
         $node
             ->children()
                 ->arrayNode('drivers')
+                    ->setDeprecated('sylius/grid-bundle', '2.x', 'Drivers config will be deprecated starting sylius/grid-bundle 2.x, register drivers using tagged service "sylius.grid_driver".')
```

Comment:
If I remember correctly, we'll need to set the current version when the deprecation has been created.

```suggestion
                    ->setDeprecated('sylius/grid-bundle', '1.15', 'Drivers config will be deprecated starting sylius/grid-bundle 2.x, register drivers using tagged service "sylius.grid_driver".')
```